task/control-plane-agent/src/mgs_sidecar.rs

```diff
@@ -632,17 +881,13 @@ impl SpHandler for MgsHandler {
 
     fn get_startup_options(
         &mut self,
-        _sender: SocketAddrV6,
-        _port: SpPort,
     ) -> Result<gateway_messages::StartupOptions, SpError> {
         ringbuf_entry_root!(Log::MgsMessage(MgsMessage::GetStartupOptions));
         Err(SpError::RequestUnsupportedForSp)
     }
 
     fn set_startup_options(
         &mut self,
-        _sender: SocketAddrV6,
```

Comment:
**suggestion:** Could these removals of unused parameters be done in a trivial side-PR?

Reply:
I think in general this would have made the diff easier to follow, but chained PRs on Github are hard to manage so I sympathize with Matt's motivation for not doing so.